hazelcast/src/main/java/com/hazelcast/internal/monitors/MetricsPlugin.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.monitors;
+
+import com.hazelcast.internal.metrics.MetricsRegistry;
+import com.hazelcast.internal.metrics.renderers.CommaSeparatedKeyValueProbeRenderer;
+import com.hazelcast.internal.metrics.renderers.ProbeRenderer;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.spi.impl.NodeEngineImpl;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+import static com.hazelcast.instance.GroupProperty.PERFORMANCE_MONITOR_METRICS_PERIOD_SECONDS;
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+/**
+ * A {@link PerformanceMonitorPlugin} that displays the content of the {@link MetricsRegistry}.
+ */
+public class MetricsPlugin extends PerformanceMonitorPlugin {
+
+    private final MetricsRegistry metricsRegistry;
+    private final int periodSeconds;
+    private final ILogger logger;
+    private final NodeEngineImpl nodeEngine;
+    private ProbeRenderer probeRenderer;
+
+    public MetricsPlugin(NodeEngineImpl nodeEngine) {
+        this.nodeEngine = nodeEngine;
+        this.metricsRegistry = nodeEngine.getMetricsRegistry();
+        this.periodSeconds = nodeEngine.getGroupProperties().getInteger(PERFORMANCE_MONITOR_METRICS_PERIOD_SECONDS);
```

Comment:
You could use `getMillis()` instead of `getInteger()` here, so you don't have to convert the value later.